src/info/LiquidatorInfo.sol

```diff
@@ -127,16 +127,29 @@ contract LiquidatorInfo is Math {
     function getCushionInfo(uint cdp, address me, uint numMembers) public view returns(CushionInfo memory info) {
         CdpDataVars memory c;
         (c.cdpArt, c.cushion, c.cdpWinners, c.bite) = pool.getCdpData(cdp);
-        info.isToppedUp = c.cushion > 0;
+        info.isToppedUp = manager.cushion(cdp) > 0;
```

Comment:
in general, we can get into a state where isToppedUp = false, and shouldCallUntop=true.
Which might sounds a bit inconsistent (how can you untop if not topped up).
we should consider setting isToppedUp to true in this case.

Also there is another issue of what happened if CDP was only partially bite. But this is an issue already in pool.sol level. So maybe let's ignore it for now.